### ISSUE TITLE
Wrong Classification of all videos as Violent

### ISSUE THREAD

**Issue by bpahuja** [JoshuaPiinRueyPan/ViolenceDetection#14]
After training the model on Hockey+Movie dataset the classifier works good on dataset videos but classifies all other random videos as violent. What do you think might be the issue ??

**Comment by ghost** [comment]
> After training the model on Hockey+Movie dataset the classifier works good on dataset videos but classifies all other random videos as violent. What do you think might be the issue ??

Can you  share me other videos as violent?

**Comment by JoshuaPiinRueyPan** [comment]
This means you should enrich your dataset.  The reason that I only use the Hockey dataset is to compare the performance of the network with the previous works.  If you want to deploy it to the real world, you must collect as much data as you can.  This may be the aka problem of the artificial neural network...
  Moreover, if you found the training loss can't get down to what you expected after you collecting many data, you can add more layer to gain the capability of the network.

**Comment by bpahuja** [comment]
thank you so much
